mbl-core/tests/partitions/test_partitions.py

```diff
@@ -86,38 +86,39 @@ def get_device_file_for_parts():
 
     This function assumes that all of the partitions are on the same device.
     """
-    return subprocess.check_output(["blkid", "-L", "rootfs1"]).rsplit(b"p", 1)[
-        0
-    ]
+    return str(
+        subprocess.check_output(["blkid", "-L", "rootfs1"])
+        .rsplit(b"p", 1)[0]
+        .split(b"/")[2],
```

Comment:
This `split` is essentially doing a `basename`, right? It's a bit non-obvious. Can you do something like this instead:
```Python
return str(
        pathlib.Path(subprocess.check_output(["blkid", "-L", "rootfs1"], text=True)).name.rsplit("p", 1)[0]
```

And can you change the name of the function to `get_device_filename_for_parts` and change the docstring to say that it's returning a filename rather than a path please?

Reply:
changed to use basename and to may it clear we are returning the blockname.